PyRDF/Node.py

```diff
@@ -317,3 +751,62 @@ def get_inputfiles(self):
                 return second_arg
         # RDataFrame may have been created with no input files
         return None
+
+    def _get_friend_info(self):
```

Comment:
We might need to come back here eventually and decide where to put this method, since it is related to the `TTree` data source. If we end up having headnode types per each data source, this would go in the `TTreeHeadNode`.